Makefile

```
@@ -2,16 +2,24 @@
 test:
 	go test -race ./...
 
+.PHONY: build
+build:
+	go build -o trivy-aws ./cmd/trivy-aws/main.go
```

You can reduce the size of the executable binary:
```suggestion
	go build -ldflags "-s -w" -o trivy-aws ./cmd/trivy-aws/main.go
```
```
trivy-aws: 273M -> 187M
trivy-aws.tar.gz: 112M -> 46M
```

---

Nice, updated.